transcriber/utils.py

```diff
@@ -117,13 +120,56 @@ def transcribe(inputFile, outputFile, language):
             rec_results.append(rec.Result())
 
     rec_results.append(rec.FinalResult())
-    vtt = write_webvtt_captions(rec_results)
-
+    print('Finished transcribing...')
+    print('Start punctuating...')
+    if punc:
+        # Punctuation
+        # Load text from json
+        text = ''
+        for i, rec_result in enumerate(rec_results):
+            result = json.loads(rec_result).get('result')
+            if not result:
+                continue
+            text += ' '.join([entry['word'] for entry in result])
+
+        # Predicts Punctuation of text
+        # Manipulate main to be able to load model
+        old_main = sys.modules['__main__']
+        sys.modules['__main__'] = transcriber
+        predictor = CasePuncPredictor(chosen_language_dir_path + '-punctuation/checkpoint', lang="de")
```

Comment:
- This shouldn't hard-code the language to `de`
- `vosk-cli` now prefers model names over languages. Maybe it would make sense to take that on and also let users specify a punctuation model name on the command line, similar to the stt model.
- It would be nice if the model path had a fallback similar to what's currently in `vosk-cli`, checking a local path first, then falling back to a globalk path, unless the whole argument is an absulute path already